modules/apigee/variables.tf

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  description = "Project ID."
+  type        = string
+}
+
+variable "organization" {
+  description = "Apigee organization"
+  type = object({
+    display_name            = optional(string)
+    description             = optional(string, "Apigee Organization created by tf module")
```

Comment:
We typically use "Managed by the Apigee Terraform module." or "Terraform-managed." as the default for descriptions.